src/main/java/com/google/sps/servlets/DestinationDataServlet.java

```diff
@@ -44,10 +46,11 @@
   private static final String HINT3_PARAMETER = "hint3";
   private static final String OBSCURITY_PARAMETER = "obscurity";
   private static final String TAG_PARAMETER = "tag";
-  private static final String REDIRECT_URL = "/destination-data";
+  private static final String REDIRECT_URL = "/index.html";
```

Comment:
I might make this variable more specific (ex. "HOME_URL").

Reply:
+1